<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtBtreeClass</class>
 <widget class="QMainWindow" name="QtBtreeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtBtree</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="NewTree">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Tree</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Insdel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert And Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome to: Visual-B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <action name="action">
   <property name="text">
    <string>插入key值</string>
   </property>
  </action>
  <action name="action_key">
   <property name="text">
    <string>删除key值</string>
   </property>
  </action>
  <action name="action_NewBtree">
   <property name="text">
    <string>New B Tree</string>
   </property>
  </action>
  <action name="action_InsertDel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Insert &amp; Delete</string>
   </property>
  </action>
  <action name="action_DelKey">
   <property name="text">
    <string>删除key值</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
